generate_readme_tables.py

```diff
@@ -99,7 +99,11 @@ def generate_summary_table():
     total_case_catch_stats['total'] = 0
 
     full_results = recurse_dir("test-cases", 2, total_case_catch_stats, is_root = True)
+    summary_table = generate_summary_table()
 
-    print(generate_summary_table())
+    with open("resources/README.md.template", "r") as fr:
```

Comment:
Thank you for doing this @owenrumney ! This will improve this repo's maintainability.